### DISCUSSION TITLE
Should the JavaScript version of XKPASSWD be version 3?

### DISCUSSION THREAD

**Discussion by podfeet** [bartificer/xkpasswd-js#77] [General]
I noticed in the previous Perl code documentation it says XKPASSWD was at version 2.1.1 so maybe the JavaScript version should be V3?

**Comment by hepabolu** [comment]
There are arguments either way:
- if this is a continuation of the old HSXKPasswd then v3 would make sense
- this is a brand new app, in a different codebase/architecture with different functionalities (no HS anymore, bookmarkable url), so starting again with version numbers would make sense

I lean towards the latter: the app is very different except for the basic functionality, even the formal name (XKPasswd-js) is different.
@bbusschots any ideas?

**Reply by bbusschots** [reply]
I actually think the accepted config should determine the major version. We have some breaking changes planned for a future release, so that should be 3.0 IMO. So, since this is very different, I think 2.5.* for now?
